build.sbt

```diff
@@ -833,11 +830,17 @@ lazy val benchmark = projectMatrix
     smithySpecs := Seq(
       (ThisBuild / baseDirectory).value / "sampleSpecs" / "benchmark.smithy"
     ),
-    (Compile / sourceGenerators) := Seq(genSmithyScala(Compile).taskValue)
+    genSmithy(Compile)
   )
   .jvmPlatform(List(Scala213), jvmDimSettings)
   .settings(Smithy4sBuildPlugin.doNotPublishArtifact)
 
+def genSmithy(config: Configuration) = Def.settings(
+  Seq(
+    Compile / sourceGenerators := Seq(genSmithyScala(Compile).taskValue),
+    Compile / resourceGenerators := Seq(genSmithyResources(Compile).taskValue)
+  )
+)
```

Comment:
i don't think this is what is causing the CI failure, but should use `config` here in the function rather than `Compile`

Reply:
you're right, big mishap lol